backend/models/user.py

```diff
@@ -9,23 +9,37 @@ class User(db.Model):
         db.Integer,
         primary_key=True)
 
+    sub = db.Column(
```

Comment:
Could this have a comment on what sub is, as it's use case is not obvious at the first glance